scoda/static/src/sass/components/_datastories.scss

```diff
@@ -62,6 +62,7 @@ body {
 .ds-banner-logo {
```

Comment:
@PhakamaniXulu please review spacing and indentation here. Use beautify or pretty print to format.